src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,39 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static alive = [];
+  constructor(name) {
+    this.health = 100;
+    this.name = name;
+
+    Animal.alive.push(this);
+  }
+
+  die(dead) {
```

Comment:
The method name 'die' suggests it is an action that the animal itself would perform, but it is used to remove other dead animals from the alive array. Consider renaming this method to better reflect its purpose, such as 'removeDead' or 'cleanupDead'.